include/deal.II/grid/grid_generator.h

```diff
@@ -977,8 +977,15 @@ namespace GridGenerator
    * Here, the number <tt>n_cells</tt> of elements has the same meaning as in
    * GridGenerator::hyper_shell.
    *
-   * @note This function is declared to exist for triangulations of all space
-   * dimensions, but throws an error if called in 1d.
+   * The grids with 12 initial cells and 3 levels of global refinement are
+   * plotted below:
+   *
+   * @image html eccentric_hyper_shell_2D.png
+   * @image html eccentric_hyper_shell_2D.png
```

Comment:
I think that you meant to use `eccentric_hyper_shell_3D.png` here.

Reply:
Yes, thanks!